app/controllers/collections_controller.rb

```diff
@@ -790,4 +803,16 @@ def reindex_items_if_necessary(collection_results)
     EOL::Solr::CollectionItemsCoreRebuilder.reindex_collection_items_by_ids(collection_item_ids_to_reindex.uniq)
   end
 
+  def limit_collection
+     if current_user.newish? || (current_user.newish? && current_user.assistant_curator?)
```

Comment:
What if the user was new but he is approved to be master curator, admin or full one?